src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-extension.js

```diff
@@ -94,7 +94,8 @@
                             callback(data);
                         }.bind(this))
                         .fail(function(data) {
-                            app.sandbox.logger.log('data could not be loaded:', data);
+                            app.logger.log('data could not be loaded:', data);
+                            callback(null);
```

Comment:
I wondered if it's a BC or not, too. But then I see it as a bug in the first place: uncaught type exception, and no callback called ever, although this is a none-error situation. Probably in everyday usage it does not occur very often: you'd need to load settings which haven't been set ever before to have this actually kick in. Or maybe a broken backend would do it, too.